app/validators/routes_and_redirects_validator.rb

```diff
@@ -147,11 +147,37 @@ def validate(location, redirect)
 
   private
 
+    def acceptable_destination?(target)
+      target.starts_with?("/") || is_valid_govuk_campaign_url?(target)
+    end
+
+    def is_valid_govuk_campaign_url?(target)
+      uri = URI.parse(target)
+      host = uri.host
+      if (host =~ /\A.+\.campaign\.service\.gov\.uk\z/i) != nil &&
```

Comment:
or you could use === if you want the boolean. 
https://ruby-doc.org/core-2.1.1/Regexp.html#method-i-3D-3D-3D